commands/audit/sca/python/python.go

```diff
@@ -22,26 +25,27 @@ import (
 )
 
 const (
-	pythonPackageTypeIdentifier = "pypi://"
+	PythonPackageTypeIdentifier = "pypi://"
 )
 
 type AuditPython struct {
 	Server              *config.ServerDetails
 	Tool                pythonutils.PythonTool
 	RemotePypiRepo      string
 	PipRequirementsFile string
+	IsCurationCmd       bool
 }
 
-func BuildDependencyTree(auditPython *AuditPython) (dependencyTree []*xrayUtils.GraphNode, uniqueDeps []string, err error) {
-	dependenciesGraph, directDependenciesList, err := getDependencies(auditPython)
+func BuildDependencyTree(serverDetails *config.ServerDetails, tech coreutils.Technology, params xrayutils2.AuditParams) (dependencyTree []*xrayUtils.GraphNode, uniqueDeps []string, err error) {
```

Comment:
why not use `AuditPython`? 
there is no need to pass tech to here, its one of the python tools

Reply:
thats right, we need to know which one of the python tools, so the tech can be pip, pipenv, poetry

Reply:
you already had this value at AuditPython.Tool

Reply:
Fixed